VCFv4.4.draft.tex

```diff
@@ -987,8 +991,9 @@ \section{FORMAT keys used for structural variants}
 ##FORMAT=<ID=AHAP,Number=1,Type=Integer,Description="Unique identifier of ancestral haplotype">
 \end{verbatim}
 \normalsize
-These keys are analogous to GT/GQ/GL/GP and are provided for genotyping imprecise variants by copy number (either because there is an unknown number of alternate alleles or because the haplotypes cannot be determined).
-CN specifies the integer copy number of the variant in this sample.
+CN specifies the total copy number over the region defined in the $<$CNV$>$, $<$DEL$>$, $<$DUP$>$ alleles.
+When CN is present, all $<$CNV$>$, $<$DEL$>$ and $<$DUP$>$ alleles must have the same SVLEN.
```

Comment:
more specifically: `When FORMAT CN is present`

Reply:
It's already in the FORMAT CN definition, but yes, I can make that explicit.